monai/networks/nets/fcnet.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Sequence, Tuple, Union
+
+import torch
+import torch.nn as nn
+
+from monai.networks.layers.factories import Act, split_args
+
+
+class FCNet(nn.Module):
+    """Plain full-connected layer neural network using dropout and PReLU activation."""
+
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        hidden_channels: Sequence[int],
+        dropout: float = 0,
+        act: Optional[Union[Tuple, str]] = Act.PRELU,
+        bias: bool = True,
+    ) -> None:
+        """
+        Defines a network accept input with `in_channels' channels, output of `out_channels' channels, and hidden layers
+        with channels given in `hidden_channels'. If `bias' is True then linear units have a bias term.
+        """
+        super().__init__()
+        self.in_channels = in_channels
+        self.out_channels = out_channels
+        self.hidden_channels = list(hidden_channels)
+        self.dropout = dropout
+        self.flatten = nn.Flatten()
+        self.hiddens = nn.Sequential()
+
+        act_name, act_args = split_args(act)
```

Comment:
would be great to use the `ADN` layer for flexibility:
https://github.com/Project-MONAI/MONAI/blob/bd1e008c36adcca64658aa10178321d4cd6854fb/monai/networks/blocks/acti_norm.py#L20-L24

Reply:
Could you be a little more explicit on this one?

Reply:
yes, using ADN would make it more flexible with the element-wise operations. this logic
```py
if self.dropout:
    seq.add_module("dropout", nn.Dropout(self.dropout))
seq.add_module("act", self.act)
```
could be replaced by:
```py
seq.add_module('ADN', ADN(dropout=dropout_params, act=act_params, norm=norm_params, ordering=ordering))
```
with user's input:
```py
norm_params=None
dropout_params=...
act_params=...
ordering="AD"
```

Reply:
Ok, updated. Hope it's what you had in mind.

Reply:
thanks it looks good to me now